public/util/validateImage.js

```diff
@@ -1,4 +1,4 @@
-const URL_REGEX = /^https:\/\/static.guim.co.uk\/sys-images\/Guardian\/.*\.(png|jpg)$/;
+const URL_REGEX = /^https:\/\/(static.guim.co.uk\/sys-images\/Guardian|uploads.guim.co.uk)\/.*\.(png|jpg)$/;
```

Comment:
Note s3-uploader maintains the original filename, so you could get `.jpeg` files.